charts/bee/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 appVersion: latest
 name: bee
-version: 0.4.5
+version: 0.4.6
```

Comment:
Since this is contains some breaking changes after bee 0.1.0, would it make sense to set version 0.5.0 and leave 0.4.x for bee 0.1.x?

Reply:
Agreed